domain/src/test/java/com/renovavision/thecocktaildb/usecases/CocktailsData.kt

```diff
@@ -0,0 +1,73 @@
+package com.renovavision.thecocktaildb.usecases
+
+import com.renovavision.thecocktaildb.domain.entities.Cocktail
+import com.renovavision.thecocktaildb.domain.entities.CocktailDetails
+
+val cocktailsList = listOf(
+    Cocktail(
+        strDrink = "747 Drink",
+        strDrinkThumb = "i9suxb1582474926.jpg",
+        key = 178318
+    ),
+    Cocktail(
+        strDrink = "155 Belmont",
+        strDrinkThumb = "yqvvqs1475667388.jpg",
+        key = 15346
+    )
+)
+
+val cocktailDetails = CocktailDetails(
```

Comment:
@AlexGolovach Use Podam library instead of creating new fake objects manually

Reply:
Done